src/test/java/org/apache/drill_web_test_framework/web_ui/WebBrowser.java

```diff
@@ -99,6 +98,19 @@ private static WebDriver getRemoteWebDriver(String driverVersion) {
     return remoteDriver;
   }
 
+  private static String getSelenoidBrowserVersion() {
+    JSONObject selenoidStatus = new JSONObject(given()
+        .get(PropertiesConst.DRILL_HOST + ":4444/status")
+        .getBody()
+        .print());
+    LinkedList<String> versions = new LinkedList<>();
+    versions.addAll(selenoidStatus.getJSONObject("browsers").getJSONObject(PropertiesConst.DRIVER_TYPE.toLowerCase()).keySet());
```

Comment:
List has a suited constructor:
```suggestion
    LinkedList<String> versions = new LinkedList<>(
        selenoidStatus.getJSONObject("browsers")
        .getJSONObject(PropertiesConst.DRIVER_TYPE.toLowerCase())
        .keySet());
```

Reply:
Done